src/main/java/com/springboot/onlinebanking/repository/UserRepository.java

```diff
@@ -0,0 +1,7 @@
+package com.springboot.onlinebanking.repository;
+
+import com.springboot.onlinebanking.model.User;
+import org.springframework.data.jpa.repository.JpaRepository;
+
+public interface UserRepository extends JpaRepository<User, Long> {
```

Comment:
@ShemsheyMaxim Do we need to add `findByNumber`?

Reply:
done